subsys/bluetooth/services/fast_pair/fp_auth.c

```diff
@@ -6,6 +6,8 @@
 
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/conn.h>
+/* Include definitions related to SMP IO capabilities. */
+#include <host/smp.h>
```

Comment:
I would avoid using internal Bluetooth headers. Consider either:
1. defining it in this module code
or
2. moving the IO capabilities definition from the internal Zephyr header to the public one (preferable option).

Reply:
Let's define it locally for now. I will open a follow-up task to move it to public header in Zephyr (I would treat it a low-priority one)